src/conf/datatypes.h

```diff
@@ -193,6 +193,14 @@ typedef struct {
     float tiltback_variable;
     float tiltback_variable_max;
     uint16_t tiltback_variable_erpm;
+    bool tiltback_bms_enabled;
+    float tiltback_cell_lv;
+    float tiltback_cell_hv;
+    int8_t tiltback_cell_lt;
+    int8_t tiltback_cell_ht;
+    int8_t tiltback_bms_ht;
+    uint8_t tiltback_bms_msg;
+    float tiltback_cell_bal;
```

Comment:
These should go into a struct, I'm thinking like this:
```c
typedef struct {
    bool enabled;
    float cell_lv_threshold;
    float cell_hv_threshold;
    int8_t cell_ltemp_threshold;
    int8_t cell_htemp_threshold;
    int8_t bms_htemp_threshold;
    float cell_balance_diff;
} CfgBMS;

typedef struct {
    // ...
    CfgBMS bms;
    // ...
} RefloatConfig;
```

Note I removed `tiltback_bms_msg`, I think we should hardcode that.

And I don't want these in the pushback alerts section, I'd like to move towards calling these just Alerts and pushback be one of the ways for alerts to be performed, next to haptics.

I'm thinking best to put these in a separate tab called "BMS", there's enough options to warrant it.